### ISSUE TITLE
ion-tests/iontestdata/good/subfieldVarUInt.ion takes 308913ms to run

### ISSUE THREAD

**Issue by dblock** [amzn/ion-js#333]
```
>> PASS: Good tests - ion-tests/iontestdata/good/subfieldVarInt.ion (144ms)
>> PASS: Good tests - ion-tests/iontestdata/good/subfieldVarUInt.ion (308913ms)
```

That can't be right, the document isn't very big.

**Comment by dblock** [comment]
This seems to have been fixed. Maybe someone can close with a link to the actual fix, curious what the problem was?

**Comment by zslayton** [comment]
Thanks for pointing this out! Those files were removed from the skip list in PR #429 following a fix that landed in PR #424.

TL;DR: those files caused an enormous empty symbol table to be instantiated/traversed unnecessarily.
